save-backend/build.gradle.kts

```diff
@@ -19,10 +19,8 @@ openApi {
     outputFileName.set("backend-api-docs.json")
     waitTimeInSeconds.set(120)
 
-    findProperty("isExecutedOnCI")?.let {
-        tasks.named<BootRun>("bootRun") {
-            jvmArgs("-Dbackend.fileStorage.location=\${HOME}/cnb/files")
-        }
+    customBootRun {
```

Comment:
@kgevorkyan they indeed used to have broken bootRun customization mechanism, which is fixed in the new version with the new `customBootRun` property. I've tested it and it correctly adds the argument to the run command (can be verified with `:save-backend:generateApiDocs --info`)